Source/Onboarding/Steps/RoomsOnboardingStep.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  RoomsOnboardingStep.swift
+//  Planetary
+//
+//  Created by Chad Sarles on 11/1/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Logger
+import CrashReporting
+import SwiftUI
+import Secrets
+
+class RoomsOnboardingStep: OnboardingStep {
```

Comment:
It would be nice to skip this step if there aren't any rooms included in the environment. I think it would be ok to just call `next.done()` in `init` in this case.